toolkit/tools/pkg/imagecustomizerlib/cosicommon.go

```diff
@@ -25,37 +25,41 @@ type ImageBuildData struct {
 	VeritySource string
 }
 
-func convertToCosi(ic *ImageCustomizerParameters) error {
-	logger.Log.Infof("Extracting partition files")
-	outputDir := filepath.Join(ic.buildDirAbs, "cosiimages")
+func convertToCosi(buildDirAbs string, rawImageFile string, outputImageFile string,
+	partUuidToFstabEntry map[string]diskutils.FstabEntry, verityMetadata []verityDeviceMetadata,
+	osRelease string, osPackages []OsPackage, imageUuid [UuidSize]byte, imageUuidStr string,
+) error {
+	outputImageBase := strings.TrimSuffix(filepath.Base(outputImageFile), filepath.Ext(outputImageFile))
```

Comment:
To be honest, we could probably give the partition files a standard base name of `partition` instead of it being dynamic. This user isn't going to see these files anyway.